codecov.yml

```diff
@@ -0,0 +1,8 @@
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        target: 50%
+        threshold: 30%
```

Comment:
30! So lenient. Don't wanna prompt more reason about how to keep your code coverage up?